### ISSUE TITLE
Crashing on startup on 2017 iMac macOS 11.0 Beta (20A5354i)

### ISSUE THREAD

**Issue by edgelord314** [Dimillian/RedditOS#15]
terminal output when running the executable directly:

```
dyld: Symbol not found: _$s7SwiftUI12ProgressViewVyxGAA0D0AAMc
  Referenced from: /Applications/Curiosity.app/Contents/MacOS/./Curiosity
  Expected in: /System/Library/Frameworks/SwiftUI.framework/Versions/A/SwiftUI
 in /Applications/Curiosity.app/Contents/MacOS/./Curiosity
zsh: abort      ./Curiosity
```

**Comment by edgelord314** [comment]
Nevermind, didn't have XCode 12 beta installed, will likely work after I did (12GB to download wth)

**Comment by edgelord314** [comment]
Installed XCode Beta 12, still getting the same error:
Termination Reason:    DYLD, [0x4] Symbol missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Symbol not found: _$s7SwiftUI12ProgressViewVyxGAA0D0AAMc
  Referenced from: /Applications/Curiosity.app/Contents/MacOS/Curiosity
  Expected in: /System/Library/Frameworks/SwiftUI.framework/Versions/A/SwiftUI
 in /Applications/Curiosity.app/Contents/MacOS/Curiosity

**Comment by chfanghr** [comment]
I ran into this issue today. This maybe related to an issue mentioned in the [macOS Big Sur 11 Beta 6 Release Notes](https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11-beta-release-notes):

```
ProgressView generic type signature has changed, adding the currentValueLabel generic parameter. This change doesn’t require any source changes, but will cause apps compiled using an earlier beta SDK to quit unexpectedly. (63580200)

Workaround: Re-compile apps for macOS Big Sur 11 Beta 6 or later to resolve this issue.
```

**Comment by chfanghr** [comment]
BTW, I got tons of compilation errors trying to build this project in xcode 12.0 beta3:
![image](https://user-images.githubusercontent.com/40087668/93293708-2e21e480-f81b-11ea-8e20-101717609125.png)

**Comment by edgelord314** [comment]
Fixed with the new 0.1 release and in Big Sur beta 20A5374i
